apps/backend/src/apps/debank/utils/rate-limit.ts

```diff
@@ -0,0 +1,28 @@
+import { NextFunction, Request, Response } from 'express';
+import { RateLimiterRedis, RateLimiterRes } from 'rate-limiter-flexible';
+import { createClient } from 'redis';
+
+import { getRedisConfig } from '../../../config/redis';
+
+const rateLimiter = new RateLimiterRedis({
+  storeClient: createClient(getRedisConfig()),
+  keyPrefix: 'deBank-rate-limit',
+  points: 20, // 20 requests
+  duration: 3 // per 3 seconds by IP
+});
+
+const formStat = (key: string, lrRes: RateLimiterRes): string =>
+  `${key} c.:${lrRes.consumedPoints} pts, l.: ${lrRes.remainingPoints} pts. Reset in ${lrRes.msBeforeNext / 1000}s`;
+
+export const rateLimiterMiddleware = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I know that we don't have anything else right now, but could we set this rate limit "globaly", not only for debank api?  
For example, you could add a health-check endpoint and make a single rate limit for debank and this endpoint.

Reply:
we can apply this middleware for any route we want e.g. /*, /debank/*, etc, or use another instance of `RateLimiterRedis` to apply the other rate-limit policy and handling. I can try to implement this middleware as global util and use it in routes. But what config should we pass to the middleware?

Reply:
the same config as we have for `debank` right now